build-system/tasks/prepend-global/index.js

```diff
@@ -51,11 +51,12 @@ function sanityCheck(str) {
 
 /**
  * @param {string} filename
+ * @param {string} opt_local
```

Comment:
missing =

Reply:
Looks like it's missing across the entire file. I'll do this in a follow up PR next week, since the egregious bug has now been fixed :)